bin/options.js

```diff
@@ -30,8 +30,8 @@ function getOptions() {
       : process.argv[process.argv.indexOf('--opts') + 1];
 
   try {
-    const opts = fs
-      .readFileSync(optsPath, 'utf8')
+    // prettier-ignore
+    const opts = fs.readFileSync(optsPath, 'utf8')
```

Comment:
Why did you ignore prettier here?

Reply:
Because the first function call is to `fs.readFileSync()`; there is no good reason to split the module from the function call -- not to mention it looks stupid. The rest of the chain processes its result via various FP methods (indented as expected for chained methods).

Reply:
I also like that no splitting the module and function call.
However, I believe that we should minimalize inline ignore because [we decided to follow Prettier](https://github.com/mochajs/mocha/pull/3344).

Reply:
please take it out, thanks